src/AmplifyGeofenceControl/ui.ts

```diff
@@ -642,20 +647,25 @@ export function AmplifyGeofenceControlUI(
   function setGeofenceListEnabled(enabled): void {
     _checkboxAll.disabled = !enabled;
 
-    if (enabled) {
-      _addGeofencebutton.classList.remove("amplify-ctrl-noHover");
-      _geofenceList.classList.remove("amplify-ctrl-noHover");
-    } else {
-      _addGeofencebutton.classList.add("amplify-ctrl-noHover");
-      _geofenceList.classList.add("amplify-ctrl-noHover");
-    }
+    enabled
+      ? _addGeofencebutton.classList.remove("amplify-ctrl-noHover")
```

Comment:
Can you add a gif to the PR that demoes this part. It's unclear why the `_geofenceList` container is no longer needed.

Reply:
Ah this is because previously the noHover css was preventing pointer events which was also preventing scrolling. I changed the behavior here to disable each item instead of the full list so that scroll would still work. Using the same gif as the list pagination as it shows that the list can scroll now (in the demo it couldnt)
![pagination](https://user-images.githubusercontent.com/68032955/145919783-3ff1b9d2-5ba4-44b2-8a49-0a484e96a4f1.gif)